source/includes/_forecast_processes.md

```diff
@@ -65,5 +76,3 @@ This table describes the attributes available for each `process` in the `process
 __id__ | string | The `id` of the individual process.
 __name__ | string | The process name for the individual process.
 __subsystem_breakdown__ | boolean | This value indicates if the individual process is included in the [Subsystem breakdown](#subsystem-breakdown).
```

Comment:
This actually specifies if this process has its own subsystem breakdown, not whether its included in another one